textmetric/chrFpp.py

```diff
@@ -0,0 +1,175 @@
+import sys
+import math
```

Comment:
The trouble with using Popovic's code is that she has licensed the file under GPL and we license our code under Apache. We can't use her code (not even if we cite it in a right way) as the licenses are incompatible, and violating license requirements is a transgression we'd rather not do (especially the way we've taken her code right now). 
This license story is not a big deal right now, but it will be bad if we submit the replication package this way.

I think we have two options: we can either rewrite chrF++ from scratch or drop the metric from the consideration altogether. I suggest the latter option since the added value of having chrF++ is not very clear and rewriting it (and checking our chrF++ is equivalent to Popovic' one) is a hassle.

Reply:
Deleted